trillian/ctfe/ct_server/main.go

```diff
@@ -140,6 +142,23 @@ func main() {
 		http.Handle("/metrics", promhttp.Handler())
 	}
 
+	if *getSTHInterval > 0 {
+		// Regularly update the internal STH for each log so our metrics stay up-to-date with any tree head
+		// changes that are not triggered by us.
+		for _, c := range cfg {
+			ticker := time.NewTicker(*getSTHInterval)
+			go func(c *configpb.LogConfig) {
+				glog.Infof("start internal get-sth operations on log %v (%d)", c.Prefix, c.LogId)
+				for t := range ticker.C {
+					glog.V(1).Infof("tick at %v: force internal get-sth for log %v (%d)", t, c.Prefix, c.LogId)
+					if _, err := ctfe.GetTreeHead(ctx, client, c.LogId, c.Prefix); err != nil {
+						glog.Warningf("failed to retrieve tree head for log %v (%d): %v", c.Prefix, c.LogId, err)
```

Comment:
Given that the main point here is to keep another metric updated, it didn't seem worth tracking -- but I can add one if you disagree.